src/TestHarness/Harness.cs

```diff
@@ -202,7 +202,7 @@ public Harness()
             };
 
```

Comment:
I'm going to remove TrackData for now since we don't have a use case for it right now. We can add it to the whole system later if we need to